### ISSUE TITLE
Double quote " from xlsx file not shown in converted PDF file

### ISSUE THREAD

**Issue by ytotare** [aspose-free-consulting/projects#271]
Hi,

We are facing an issue while converting the xlsx file to PDF. Following is the code snippets we used to convert the file.

![code](https://user-images.githubusercontent.com/4902151/207297534-41d3cb7d-4592-4dc5-8e7b-a5cf7be9eddc.png)

converted pdf file [2.pdf](https://github.com/aspose-free-consulting/projects/files/10217064/2.pdf) does not show the double quote " for the cells which has custom cell formatting like # ##/##"

 ![formats](https://user-images.githubusercontent.com/4902151/207297462-5ce43c94-a0ab-405c-a780-8d21c9a075ad.png) 

Can you please help us resolving this issue?

Yogesh Totare

**Comment by amjad-sahi** [comment]
@ytotare,

Thanks for the screenshot and output PDF document.

You already shared your template Excel file in the forum [thread](https://forum.aspose.com/t/double-quote-from-xlsx-file-not-shown-in-converted-pdf-file/256552) for the issue. I was able to reproduce the issue as you mentioned by converting your template file to PDF file format. I found that double quotes " from XLSX file are not shown in converted PDF file. I noticed this issue with cells, E61, E62, E63, L61, L62, S61, S62, etc. on the sheet.
e.g.
**Sample code:**

```java
Workbook workbook = new Workbook("f:\\files\\2.xlsx");
workbook.save("f:\\files\\out1.pdf");
```
I logged a ticket with an id “CELLSJAVA-45026” for your issue. Once we have an update on it, we will let you know.
